backend/apis/parse-docket-pdf.utils.js

```diff
@@ -1,13 +1,20 @@
-const sectionHeaderRegex = /^([A-Z][A-Z ]+)/;
+const sectionHeaderRegex = /^([A-Z][A-Z ]+)/g;
 
 exports.parsePdfText = function parsePdfText(text) {
   const lines = text.split("\n");
   const sections = [];
+  let gotCase = 0;
 
   for (let i = 0; i < lines.length; i++) {
     const line = lines[i];
     const match = sectionHeaderRegex.exec(line);
-    if (match) {
+    if (match && match[1].trim() !== "CASE NUMBER") {
```

Comment:
What if we just avoided all duplicate sections? Do you know of any cases where we need any duplicate sections?

```js
for (let i = 0; i < lines.length; i++) {
  const line = lines[i];
  const match = sectionHeaderRegex.exec(line);
  if (match) {
    const name = match[1].trim();
    if (!sections.some(s => s.name === name) {
      sections.push({lineNumber: i, name})
    }
  }
}
```

Reply:
The file I was using for testing is pretty big (Evans 141100464) and had 20 some odd pages. The only repeating section was the CASE NUMBER. After implementing the code, I only got back the top line sections ending in PROCEEDINGS. I believe that was the desired result. No other sections  heading repeat so there are no other section duplicates. Worth writing a test to prove it?

Reply:
If case number is the only section that repeats, then okay we can only check for that section 👍 

Let's clean up your code a bit though. For example, we don't need to copy/paste the `sections.push()` or `match[1].trim()` statements.

Here's a suggested implementation that might be a bit easier to follow:

```js
let foundCaseNumberSection = false

for (let i = 0; i < lines.length; i++) {
  const line = lines[i];
  const match = sectionHeaderRegex.exec(line);
  if (match) {
    addSection(match[1], i)
  }
}

function addSection(name, lineNumber) {
  const shouldAdd = name !== 'CASE NUMBER' || !foundCaseNumberSection
  if (name === 'CASE NUMBER') {
    foundCaseNumberSection = true
  }
  if (shouldAdd) {
    sections.push({name, lineNumber})
  }
}
```

Reply:
Do you want me to update the code with this suggestion (thanks for that) and remove the g flag from the regex?

Reply:
yep, remove the `g` flag from the regex since it's not being used or necessary (at least for now)